scripts/download.sh

```diff
@@ -4,24 +4,44 @@
 
 install_dir="${data_dir}/installation_files"
 
-fetch_and_download_latest_images() {
+fetch_latest_images() {
    index_url=${1}
    latest_images="$(fetch "${URL}""${index_url}" | jq '.images |  map(select(.type == "full")) | sort_by(.version) | .[-1]'))"
    downloadable_files="$(echo "${latest_images}" | jq -r '.files[].path')"
 
-   echo "${downloadable_files}" | while read -r files; do
+   ret=$(echo "${downloadable_files}")
+   if [ "${ret}" != "" ]; then
+     echo ${ret}
+   fi
+}
+
+download_latest_images() {
+   index_url=${1}
+   images=$(fetch_latest_images "${index_url}")
+
+   for files in "${images}"; do
       file=$(basename "${files}")
-      fetch_persistent_files "${URL}${files}" "${install_dir}/${file}"
-      echo "update $file $file.asc" >> "${install_dir}/ubuntu_command"
+      wget "${URL}${files}"
```

Comment:
You could have used the same logic that's in `fetch` to choose between `wget` and `curl`